### PR TITLE
DAT-2970

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/DAT-2970

The infobox image always will have the width of the screen (will be stretched)

### CODE REVIEW

**Comment by RafLeszczynski** [comment]
LGTM

File: front/scripts/main/components/ImageMediaComponent.ts [hunk 1]
```diff
@@ -53,43 +53,39 @@ App.ImageMediaComponent = App.MediaComponent.extend(App.ArticleContentMixin, {
...
-	 *   mode: string crop mode
-	 *   height: number height of image
-	 *   width: number width of image
-	 * }
+	 * @return ThumbnailerParams
 	 */
-	infoboxImageParams: Em.computed('media', 'articleContent.width', 'computedHeight', 'imageAspectRatio', {
-		get(): thumbnailerParams {
+	infoboxImageParams: Em.computed('media', 'imageAspectRatio', {
+        get(): ThumbnailerParams {
```
Comment (rogatty) [inline front/scripts/main/components/ImageMediaComponent.ts]: Use tabs, not spaces.

Comment (dianafa) [inline front/scripts/main/components/ImageMediaComponent.ts]: I do. Strange

File: front/scripts/main/components/ImageMediaComponent.ts [hunk 2]
```diff
@@ -53,43 +53,39 @@ App.ImageMediaComponent = App.MediaComponent.extend(App.ArticleContentMixin, {
...
-		get(): thumbnailerParams {
+	infoboxImageParams: Em.computed('media', 'imageAspectRatio', {
+		get(): ThumbnailerParams {
 			var media: ArticleMedia = this.get('media'),
-				articleContentWidth: number = this.get('articleContent.width'),
-				computedHeight: number = this.get('computedHeight'),
 				imageAspectRatio: number = this.get('imageAspectRatio'),
-				maximalWidth: number = Math.floor(media.height * imageAspectRatio);
+				maximalWidth: number = Math.floor(media.height * imageAspectRatio),
+				windowWidth: number = window.innerWidth;
```
Comment (rogatty) [inline front/scripts/main/components/ImageMediaComponent.ts]: How about using `ViewportMixin` instead? It's a little bit more clever: `this.set('viewportDimensions.width', Math.max(document.documentElement.clientWidth, window.innerWidth || 0));`

Comment (dianafa) [inline front/scripts/main/components/ImageMediaComponent.ts]: Thank you! :rose:

**Pull request merged**